### PR TITLE
seq2seq tutorial example not working for python 3.4/3.5 (#5118)

### PR DESCRIPTION
**Pull request by Invisibility**
This is a fix for issue #5118

### CODE REVIEW

**Comment by caisq** [comment]
Can you sign the CLA please, @Invisibility ?

**Comment by Invisibility** [comment]
@caisq I did sign the Google Individual CLA (based on the instructions at https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md) on Nov 20 with my github account name Invisibility. Let me know if I need to do anything extra.

File: tensorflow/models/rnn/translate/data_utils.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def data_to_token_ids(data_path, target_path, vocabulary_path,
           counter += 1
           if counter % 100000 == 0:
             print("  tokenizing line %d" % counter)
-          token_ids = sentence_to_token_ids(line, vocab, tokenizer,
+          token_ids = sentence_to_token_ids(tf.compat.as_bytes(line), vocab, tokenizer,
```
Comment (girving) [inline tensorflow/models/rnn/translate/data_utils.py]: Fix wrapping to 80 columns.

**Review by girving — CHANGES_REQUESTED** [review]

**Review by girving — APPROVED** [review]

**Comment by girving** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
Macos tests fixed. 
Jenkins, test this please.

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by caisq** [comment]
@tensorflow-jenkins , test this please.

**Comment by caisq** [comment]
The failure of //tensorflow/core:gpu_tracer_test in the Linux GPU build is unrelated. Merging this PR now.

Thanks, @Invisibility

**Pull request merged**